src/fmu/sumo/explorer/_child_object.py

```diff
@@ -27,22 +41,58 @@ def __init__(self, sumo_client, meta_data):
 
     @property
     def blob(self):
+        """Returns the __blob attribute"""
         if self.__blob is None:
             self.__blob = self.__get_blob()
 
         return self.__blob
 
     def __get_blob(self):
+        """Gets the object from blob store"""
         blob = self.sumo.get(f"/objects('{self.sumo_id}')/blob")
         return blob
 
     @property
     def png(self):
+        """returns the __png attribute"""
         if self.__png is None:
             self.__png = self.__get_png()
 
         return self.__png
 
     def __get_png(self):
+        """Gets the object as png"""
         png = self.sumo.get(f"/objects('{self.sumo_id}')/blob", encoder="png")
-        return png
\ No newline at end of file
+        return png
+
+
+class SurfaceObjects:
+
+    """Container for a set of objects"""
+
+    def __init__(object_ids, case):
+        """Init of objects
+        args:
+        object_ids (dict): key is real, value is object id
```

Comment:
Design-discussion probably, and we can come back to it, but this limits the collection to one per realisation. (Key in dict is real(ization)?)  Are there use cases where this breaks down? Does it make sense to initialize this with object ID's only - not require it to be a dict with realisations as well?

Reply:
Oh yes, this was just something I added, this is far from properly thought through. Again, we need a discussion. But this one could be something to postpone for later. Conceptually it would be nice with a parent class, Like ObjectCollection or summat, and then SurfaceCollection, and TableCollection could be children of this one etc